### PR TITLE
implementing requirements, more details in description

### PR DESCRIPTION
**Pull request by KarolisZiukas**
Indexed properties (only for demo, not useful for now)
Exceptions for file IO
string.empty for empty strings
Extracted strings to separate strings class
Fixed some error with properties
Created unit test, but not yet pushed

### CODE REVIEW

File: BlueTeamFigures.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RedBallTracker
+{
+    class BlueTeamFigures
+    {
+        //TODO Karolis Indexed properties for demo purposes, not really useful
+        private string[] figuresNames = new string[5]
```
Comment (Adomas202) [inline BlueTeamFigures.cs:12]: This is class properties for getters and setters? @droplt ?
Are these names just for sake of implementation? :D

File: Strings.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RedBallTracker
+{
+    public static class Strings
```
Comment (Adomas202) [inline Strings.cs]: Maybe this class could be called constants and also hold other types constants if needed?

**Review by Adomas202 — APPROVED** [review]
Really nice work updating a ton of these minor and bigger things that Donatas asked us to do. I still have a couple of questions tho 👍

File: Strings.cs (hunk 2 above)
Comment (KarolisZiukas) [reply Strings.cs]: That's actually a great idea, I will change this

File: BlueTeamFigures.cs (hunk 1 above)
Comment (KarolisZiukas) [reply BlueTeamFigures.cs:12]: These names is just for implementation, to show how indexed properties work ;)

**Pull request merged**